cafebabel/static/js/edit-article.js

```diff
@@ -89,6 +89,6 @@ window.addEventListener('load', addTagsRemoveListener)
 function displayTags(tags) {
   const container = tags.list.cloneNode(false)
   tags.list.replaceWith(tags._createTagsList(container, tags.values))
-  document.querySelector('.tags ul li:last-child').classList.add('fadeIn')
+  document.querySelector('.tags .tags-list li:last-child').classList.add('fadeIn')
```

Comment:
Not sure why adding yet another class within such a specific `.tags` class

Reply:
@vinyll - @johan creates another -ul- for completion, is your comment stays relevant with this new context?

Reply:
Johan != @johan on github, FYI, @pointbar. :) / disables notifications.